docker/base.Dockerfile

```diff
@@ -2,75 +2,60 @@ FROM ubuntu:jammy
 LABEL maintainer="IETF Tools Team <tools-discuss@ietf.org>"
 
 ENV DEBIAN_FRONTEND noninteractive
-
-RUN apt-get update --fix-missing
+WORKDIR /root
 
 # Install dependencies
-RUN apt-get install -y --fix-missing \
-    libpango-1.0-0 \
-    libssl-dev \
-    fontconfig \
-    libharfbuzz0b \
-    libpangoft2-1.0-0 \
-    pkg-config \
-    libxml2-utils \
-    groff \
-    wget \
-    unzip \
-    locales
-
-# Install Python 3.10
-RUN apt-get install -y \
-    python3.10 \
-    python3.10-dev \
-    python3-pip \
-    python3.10-distutils
+RUN apt-get update --fix-missing && \
+    apt-get install -y --fix-missing \
+        libpango-1.0-0 \
+        libssl-dev \
+        fontconfig \
+        libharfbuzz0b \
+        libpangoft2-1.0-0 \
+        pkg-config \
+        libxml2-utils \
+        groff \
+        wget \
+        unzip \
+        locales \
+        python3.10 \
+        python3.10-dev \
+        python3-pip \
+        python3.10-distutils && \
+    locale-gen en_US.UTF-8
```

Comment:
To remove cached and temporary files, add:
```
 && rm -rf /var/lib/apt/lists/* /var/log/dpkg.log \
 && apt-get autoremove -y && apt-get clean -y
```

Reply:
I've added these commands to both base and dev Dockerfiles.